cli/exp_task_logs.go

```diff
@@ -71,8 +89,154 @@ func (r *RootCmd) taskLogs() *serpent.Command {
 			_, _ = fmt.Fprintln(inv.Stdout, out)
 			return nil
 		},
+		Options: serpent.OptionSet{
+			{
+				Flag:          "follow",
+				FlagShorthand: "f",
+				Description:   "Follow logs in real-time, similar to 'tail -f'.",
+				Value:         serpent.BoolOf(&follow),
+			},
+		},
 	}
 
 	formatter.AttachOptions(&cmd.Options)
 	return cmd
 }
+
+// TODO(cian): this will need to be updated when the task data model is updated.
+func (*RootCmd) followTaskLogs(inv *serpent.Invocation, client *codersdk.Client, taskID uuid.UUID) error {
```

Comment:
Should we have this functionality in coderd instead?

Reply:
Driving by randomly to say it could be useful in coderd so we could use it with remote MCP (and I suppose the plugins and frontend could also make use of it).  Would have nice parity with the other log endpoints like the agent and build logs.